generator/index.js

```diff
@@ -772,6 +779,47 @@ const getAllFilesInDirStructure = (module.exports.replaceInFile = async (srcPath
   }
 });
 
+// utility function used to extend package.json with the dependencies from custom template's package.json
+const extendPackageJSONWithCustomTemplatesDependencies = (module.exports.extendPackageJSONWithCustomTemplatesDependencies = async (api, srcPathPrefix) => {
+  try {
+    const packageJSONPath = path.join(srcPathPrefix, 'package.json');
+    if (!fs.existsSync(packageJSONPath)) {
+      return;
```

Comment:
We can return an informative console text about path not exist.

Reply:
done